app/models/company.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Model that allows storing companies
+class Company < ApplicationRecord
+  before_save :ensure_only_one_selected
+
+  validates :name, :website, :selected, presence: true
+
+  has_many :social
+
+  scope :is_selected, -> { find_by(selected: true) }
```

Comment:
```suggestion
  scope :selected, -> { find_by(selected: true) }
```

Rubyist don't use `is` a lot.